test/ex_doc/retriever_test/Elixir.ExDoc.RetrieverTest.ModuleNotCompiledWithFlagDocs.erl

```diff
@@ -0,0 +1,4 @@
+% This Erlang module is deliberately named like an Elixir module in order to
+% trigger a warning.  See https://github.com/elixir-lang/ex_doc/issues/453
+
+-module('Elixir.ExDoc.RetrieverTest.ModuleNotCompiledWithFlagDocs').
```

Comment:
This looks like a fixture, right? Is there any issue to put this file into `test/fixtures`?